src/assets/i18n/en.json

```diff
@@ -0,0 +1,57 @@
+{
+  "NETWORK_OFFLINE": "You are offline, please connect to a network",
+  "PROFILE_UPDATE_SUCCESS": "Profile Updated Successfully",
+  "PROFILE_LOAD_ERROR": "Error loading profile data. Please try again later.",
+  "IMAGE_UPLOAD_SUCCESS": "Image uploaded successfully",
+  "IMAGE_UPLOAD_FAILED": "Upload failed",
+  "FORM_REQUIRED_FIELDS_ERROR": "Please fill out all the required fields",
+  "LINK_COPY_SUCCESS": "Link copied to clipboard",
+  "DOWNLOAD_FAILED": "Downloading failed !!",
+  "REPORT_SHARE_FAILED": "Failed to fetch report data for sharing.",
+  "SOMETHING_WRONG": "Something went wrong",
+  "CAMERA_PERMISSION_REQUIRED": "Camera permission is required to scan QR code. Please enable it in your browser or app settings.",
+  "CAMERA_PERMISSION_CHECK_ERROR": "Error checking camera permissions.",
+  "LINK_INVALID_ERROR": "Invalid Link, please try with another link",
+  "FORM_LOAD_ERROR": "Error loading form JSON",
+  "PROFILE_DATA_LOAD_ERROR": "Error loading profile data",
+  "PROFILE_PHOTO_REMOVED": "Removed current photo",
+  "PROFILE_IMAGE_ATTACHED": "Successfully Attached",
+  "PROFILE_IMAGE_UPLOAD": "Please upload an image file",
+  "PROJECT_CERTIFICATE": "Projects Certificate",
+  "VIEW": "View",
+  "PROFILE": "Profile",
+  "LOGOUT": "Logout",
+  "CERTIFICATE": "Certificate",
+  "LOAD_MORE": "Load More",
+  "PROFILE_DETAILS": "Profile Details",
+  "EDIT_PROFILE": "Edit Profile",
+  "PROFILE_EDIT": "Profile Edit",
+  "PROFILE_UPDATE": "Update Profile",
+  "PROJECT_CATEGORY": "Project Categories",
+  "PROJECT_UNDER": "Projects under",
+  "PROJECT_REPORT": "Project Reports",
+  "PROJECT_NO_DATA": "No Data available",
+  "REPORT_NO_DATA": "Sorry there is no data available for the entity you selected",
+  "OK": "Ok",
+  "SCAN_QR_CODE": "Scan QR Code",
+  "DEEP_LINK_REDIRECTING": "Redirecting....",
+  "CERTIFICATE_VERIFIED": "Certificate verified",
+  "CERTIFICATE_ISSUED": "Certificate issued to",
+  "CERTIFICATE_SUCCESS": "For successfully completing the project",
+  "CERTIFICATE_DATE_ON": "On Elevate on",
+  "OKAY": "OKAY",
```

Comment:
Keep the value to "Okay" and if needed capital add css style where ever necessary